docs/en/observability/whats-new.asciidoc

```diff
@@ -3,21 +3,55 @@
 
 Here are the highlights of what's new and improved in {minor-version}.
 
-coming[8.0.0]
-
-// Other versions:
-// {observability-guide-all}/7.12/whats-new.html[7.12] |
-// {observability-guide-all}/7.11/whats-new.html[7.11] |
-// {observability-guide-all}/7.10/whats-new.html[7.10] |
-// https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
-// https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
-// https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
-// https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
-// https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
-// https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
+Other versions:
+{observability-guide-all}/8.1/whats-new.html[8.1] |
+{observability-guide-all}/8.0/whats-new.html[8.0] |
+{observability-guide-all}/7.17/whats-new.html[7.17] |
 
 // tag::whats-new[]
+[discrete]
+== Fleet now works with limited privileges
 
-// What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
+Fleet and Elastic Agent Integrations previously required a superuser role.
+This required organizations to give users very broad access,
+which created more potential for security and operational risks.
+
+You can now grant users a more limited set of Kibana privileges for Fleet and Integrations.
+This will allow you to safely grant access to more users within your organization.
+No Elasticsearch privileges are required since Kibana handles updating Elasticsearch assets automatically.
```

Comment:
```suggestion
Starting in 8.1, you can grant users a more limited set of Kibana privileges for Fleet and Integrations.
No Elasticsearch privileges are required since Kibana updates Elasticsearch assets automatically.
This will allow you to safely grant access to more users within your organization.
The required Kibana privileges are `All` for the Fleet and Integrations feature.
```